live-examples/css-examples/filter-effects/sepia.html

```diff
@@ -1,20 +1,27 @@
 <section id="example-choice-list" class="example-choice-list large" data-property="filter">
+<div class="example-choice">
+<pre><code id="example_one" class="language-css">filter: sepia(0);</code></pre>
+<button type="button" class="copy hidden" aria-hidden="true">
+    <span class="visually-hidden">Copy to Clipboard</span>
+</button>
+</div>
+
 <div class="example-choice" initial-choice="true">
-<pre><code id="example_one" class="language-css">filter: sepia(60%);</code></pre>
+<pre><code id="example_one" class="language-css">filter: sepia(0.20);</code></pre>
```

Comment:
...and one more. We should try to generate these IDs, too, it is easy to get them wrong.

Reply:
We could [lint for duplicate ids](https://github.com/htmllint/htmllint/wiki/Options#id-no-dup) in with TravisCI to *reduce* the number of issues in the future.

Reply:
Actually, does the editor even use IDs? That's probably something I should've check when removing clipboard targets but I assumed they were used elsewhere. Removing them doesn't seem to change anything and I can't find anything in the code.

Reply:
I don't know :). @schalkneethling probably does though.